plotting.py

```diff
@@ -2,11 +2,11 @@
 import utils
 import itertools
 import collections
-from giotto.mapper import visualization
+from gtda.mapper import visualization
```

Comment:
I'd prefer "from gtda.mapper import plot_static_mapper_graph" (plus any other if needed, e.g. the interactive one)

Reply:
I applied the changes to the notebook as well.